WalletWasabi.Gui/Tabs/WalletManager/GenerateWalletSuccessView.xaml

```diff
@@ -11,7 +11,14 @@
     <TextBlock Text="You can recover your wallet on any computer with:" FontWeight="Bold" />
     <TextBlock Text="- your mnemonic words AND" FontWeight="Bold" />
     <TextBlock Text="- your password" FontWeight="Bold" />
-    <TextBlock Text="Unlike most other wallets if an attacker aquires your mnemonic words, it will not be able to hack your wallet without knowing your password. However, unlike other wallets, you will not be able to recover your wallet only with your mnemonic words if you lose your password." TextWrapping="Wrap" />
+    <TextBlock Text="Unlike most other wallets if an attacker aquires your mnemonic words, it will not be able to hack your wallet without knowing your password. However, unlike other wallets, you will not be able to recover your wallet only with your mnemonic words if you lose your password. For that reason, verify the password is correct and back it up." TextWrapping="Wrap" />
```

Comment:
I should write something like this:
Backup your password too! Unlike most other wallets if an attacker acquires your mnemonic words, it will not be able to hack your wallet without knowing your password. Only with the mnemonic words you will not be able to recover your wallet! For that reason, verify the password is correct and back it up.

Reply:
Too long, don't read. This is true for the original text, too.

Reply:
Agree. What do we do with that text? Should I rollback to the original text?

Reply:
Not sure at this point, I will look at the new UX workflow after the functionality is working.